src/ui/VoiceMessageInput/types.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-redeclare */
```

Comment:
It's because we have duplicated declarations here like
```
export const VoiceMessageInputStatus = 
export type VoiceMessageInputStatus =
```

There are several places we're doing like this. I disabled the rule for them for now but I guess we'd better give different name for variable & type next time. It's a bit hard to differenciate  which one is variable or type.

Reply:
I wanted different name for this ~ 
`VoiceMessageInputStatus` & `VoiceMessageInputStatusType` cc @HoonBaek

Reply:
Ah this is my legacy..

Reply:
Oh then better to give them different names and remove `eslint-disable` too later.